drivers/bridge/port_mapping.go

```diff
@@ -49,8 +49,15 @@ func (n *bridgeNetwork) allocatePortsInternal(bindings []types.PortBinding, cont
 			}
 			bs = append(bs, bIPv4)
 		}
+
 		// Allocate IPv6 Port mappings
-		if ok := n.validatePortBindingIPv6(&bIPv6, containerIPv6, defHostIP); ok {
+		// If the container has no IPv6 address, allow proxying host IPv6 traffic to it
+		// by setting up the binding with the IPv4 interface if the userland proxy is enabled
```

Comment:
Its worth adding a comment as to *why* we are doing this.

Reply:
Ah, yes, agreed @arkodg perhaps you could add some description